### PR TITLE
Add task name to Tasks Runs page

### LINKED ISSUE #13046: Task name should be on the runs page
as a user
with admin
when i view the list of task runs
then i would like to know which task i am viewing the runs for.

let's add the task name to the runs page of a task.

### PR DESCRIPTION
**Pull request by Palakp41**
Closes #13046 

_Briefly describe your proposed changes:_

  - [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [ ] http/swagger.yml updated (if modified Go structs or API)
  - [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/tasks/components/TaskRunsPage.tsx [hunk 1]
```diff
@@ -49,7 +51,9 @@ class TaskRunsPage extends PureComponent<Props> {
         <Page titleTag="Runs">
           <Page.Header fullWidth={false}>
             <Page.Header.Left>
-              <PageTitleWithOrg title="Runs" />
+              <PageTitleWithOrg
+                title={`${_.get(currentTask, 'name', '')} - Runs`}
```
Comment (chnn) [inline ui/src/tasks/components/TaskRunsPage.tsx]: Should the `- ` be there if the name lookup fails? Might look weird having ` - Runs` as a title.

File: ui/src/tasks/actions/index.ts [hunk 2]
```diff
@@ -458,7 +458,7 @@ export const importTask = (script: string) => async (
 export const getRuns = (taskID: string) => async (dispatch): Promise<void> => {
   try {
     dispatch(setRuns([], RemoteDataState.Loading))
-
+    await dispatch(selectTaskByID(taskID))
```
Comment (chnn) [inline ui/src/tasks/actions/index.ts]: Could these be done in parallel?
```
const [runs] = await Promise.all([
  client.tasks.getRunsByTaskID(taskID)
  dispatch(selectTaskByID(taskID))
])
```

File: ui/src/tasks/components/TaskRunsPage.tsx (hunk 1 above)
Comment (Palakp41) [reply ui/src/tasks/components/TaskRunsPage.tsx]: I updated it with a getter for title

File: ui/src/tasks/actions/index.ts (hunk 2 above)
Comment (Palakp41) [reply ui/src/tasks/actions/index.ts]: Yes, they don't depend on each other, I updated it

**Pull request merged**